src/Extensions.Base/HostExtension.h

```diff
@@ -30,8 +30,8 @@ class ATL_NO_VTABLE CHostExtension :
 
 	DECLARE_PROTECT_FINAL_CONSTRUCT()
 
-	HRESULT FinalConstruct() { return S_OK; }
-	void FinalRelease() {}
+	static HRESULT FinalConstruct() { return S_OK; }
+	static void FinalRelease() {}
```

Comment:
Make sure that this isn't part of a public interface. It looks like it might be. If yes, just disable the warnings for this file.